contents/docs/cdp/batch-exports/index.md

```diff
@@ -117,3 +117,11 @@ sequenceDiagram
 
 - Some features of the old export destinations are still being ported over.
   - This includes logs and error reporting.
+ 
+## How can I export person info?
+
+Currently it's not possible, but it's on the roadmap:
+- [person data export](https://github.com/PostHog/posthog/issues/20339).
+- Export person properties on events (`event.person_properties`) with [custom schemas](https://github.com/PostHog/posthog/issues/20367).
+
+Note: Our implementation of person ids on events (`event.person_id`) is eventually consistent, which for batch export destinations means joining event `distinct_id` with the person data export.
```

Comment:
I would omit this. It's not wrong it's just that we don't have a person data export (as stated above). Eventually, once we develop the person model this will be the way to go. As it stands, this note seems confusing.

Reply:
I need it to answer why we're not exporting `person_id` as that's the user question